release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1579,6 +1584,18 @@ The issue might still occur with some combinations of input for isolated CPUs in
 [id="ocp-release-note-observability-bug-fixes_{context}"]
 ==== Observability
 
+[discrete]
+[id="ocp-release-note-oc-mirror-bug-fixes_{context}"]
+==== oc mirror
+
+* Previously, when using `oc-mirror --v2 delete --generate`command, the contents of the working-dir/cluster-resources folder were cleared. With this fix, the working-dir/cluster-resources folder is not cleaned when the delete feature is used. (link:https://issues.redhat.com/browse/OCPBUGS-48430[*OCPBUGS-48430*])
+
+* Previously, release images are now signed using a new SHA-256 trusted key instead of the previously used SHA-1 key. On {op-system-base} 9 FIPS STIG-compliant machines, verification of release signatures using the old SHA-1 key failed due to security restrictions on weak keys. This update ensures that oc-mirror now uses the SHA-256 key for release signature verification. (link:https://issues.redhat.com/browse/OCPBUGS-48314[*OCPBUGS-48314*])
```

Comment:
Change to Previously, release images were signed using a SHA-1 key. Now they are signed using a new SHA-256 trusted key.